src/lib/components/labelCard.svelte

```diff
@@ -1,10 +1,14 @@
 <script lang="ts">
+    import { app } from '$lib/stores/app';
+    import { base } from '$app/paths';
+
     export let name: string;
     export let group: string;
     export let value: string | number | boolean;
     export let disabled = false;
     export let padding = 1;
     export let icon: string = null;
+    export let imageIcon: string = null;
```

Comment:
I had to update the label card component because it didn't accept image icons. Is that okay?

Reply:
Torsten: for now, let's go with this.